src/Core/AdminConsole/Repositories/IOrganizationUserRepository.cs

```diff
@@ -58,4 +58,6 @@ UpdateEncryptedDataForKeyRotation UpdateForKeyRotation(Guid userId,
     /// Returns a list of OrganizationUsers with email domains that match one of the Organization's claimed domains.
     /// </summary>
     Task<ICollection<OrganizationUser>> GetManyByOrganizationWithClaimedDomainsAsync(Guid organizationId);
+
+    Task RevokeOrganizationUserAsync(IEnumerable<Guid> userIds);
```

Comment:
Please add xmldoc here. Also, I would suggest renaming this to fit in with our usual naming scheme:
```suggestion
    Task RevokeManyByUserIdAsync(IEnumerable<Guid> userIds);
```